extras/indexing/src/main/java/org/apache/rya/api/client/mongo/MongoInstall.java

```diff
@@ -191,26 +192,22 @@ private static MongoDBRdfConfiguration makeRyaConfig(
 
         final MongoDBRdfConfiguration conf = connectionDetails.build(ryaDetails.getRyaInstanceName());
 
-        // The Mongo implementation of Rya does not currently support PCJs.
-        if(ryaDetails.getPCJIndexDetails().isEnabled()) {
-            log.warn("The install configuration says to enable PCJ indexing, but Mongo RYA does not support that " +
-                    "feature. Ignoring this configuration.");
-        }
-        conf.set(ConfigUtils.USE_PCJ, "false");
+        conf.setBoolean(ConfigUtils.USE_PCJ, ryaDetails.getPCJIndexDetails().isEnabled());
 
         // Mongo does not support entity indexing.
         if(ryaDetails.getEntityCentricIndexDetails().isEnabled()) {
             log.warn("The install configuration says to enable Entity Centric indexing, but Mongo RYA does not support " +
                     "that feature. Ignoring this configuration.");
         }
-        conf.set(ConfigUtils.USE_ENTITY, "false");
+        //TODO mongo now has an entity index, just needs CLI support.
```

Comment:
Address the TODOs and out of date comments in there. Should figure out if we're doing entity centric stuff in this ticket or not.

Reply:
discussed in person